starter-code/client/src/app/services/journal-entries.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { Http, Response } from '@angular/http';
+
+@Injectable()
+export class JournalEntriesService {
+
+  BASE_URL: string = 'http://localhost:3000';
```

Comment:
since you are doing request to `http://localhost:3000/api` you could add the word `api` on `BASE_URL`